include/stl2/detail/view/view_closure.hpp

```diff
@@ -23,7 +22,7 @@ STL2_OPEN_NAMESPACE {
 
 		template <class T>
 		concept bool Pipeable =
-			std::is_base_of_v<__pipeable_base, T> &&
+			DerivedFrom<T, __pipeable_base> &&
```

Comment:
Again, I don't think we want "publicly and unambiguously".

Reply:
Again, I do ;)